sdk/python/kfp/compiler/testdata/v2_compatible_two_step_pipeline.yaml

```diff
@@ -0,0 +1,265 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: my-test-pipeline-
+  annotations:
+    pipelines.kubeflow.org/kfp_sdk_version: 1.4.1
+    pipelines.kubeflow.org/pipeline_compilation_time: '2021-03-08T06:53:16.328199'
+    pipelines.kubeflow.org/pipeline_spec: '{"inputs": [{"default": "gs://output-directory/v2-artifacts",
+      "name": "pipeline-output-directory"}, {"default": "my-test-pipeline", "name":
+      "pipeline-name"}], "name": "my-test-pipeline"}'
+    pipelines.kubeflow.org/v2_pipeline: "true"
+  labels: {pipelines.kubeflow.org/kfp_sdk_version: 1.4.1}
```

Comment:
Do we need the `pipelines.kubeflow.org/kfp_sdk_version: 1.4.1` in both annotations and labels?
Note that this need to be updated after loading golden, otherwise the next KFP sdk version bump will break the test. Here's an similar example in v2 tests:
https://github.com/kubeflow/pipelines/blob/fc7afdd0a07c328584cc6c75c8a7e0c04664c5ea/sdk/python/kfp/v2/compiler_cli_tests/compiler_cli_tests.py#L38-L40

But feel free to leave a `TODO(chensun): ...` in the test code and I can address this later.

Reply:
I deleted the `metadata` section before comparing them so I think this works regardless of SDK version. Let me know if this still won't work.

Reply:
I see. I missed that part. This works then.